src/Foundatio.TestHarness/Messaging/MessageBusTestBase.cs

```diff
@@ -10,6 +10,7 @@
 using Foundatio.Utility;
 using Xunit;
 using Foundatio.AsyncEx;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
```

Comment:
Is there any reason this is needed?

Reply:
Nope. I copied the code from another project. Maybe Resharper Added this reference.